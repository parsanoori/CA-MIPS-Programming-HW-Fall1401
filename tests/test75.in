18
801842643
655923826
653175700
566510672
628532802
320785475
153244680
236093220
947183953
221335252
277757629
953238980
821705098
439675356
419511306
877291583
176189606
973823901