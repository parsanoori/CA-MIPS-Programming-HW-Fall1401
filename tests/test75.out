153244680
176189606
221335252
236093220
277757629
320785475
419511306
439675356
566510672
628532802
653175700
655923826
801842643
821705098
877291583
947183953
953238980
973823901